sdk/src/types/evaluation.d.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable */
+/**
+ * This file was automatically generated by json-schema-to-typescript.
+ * DO NOT MODIFY IT BY HAND. Instead, modify the source JSONSchema file,
+ * and run json-schema-to-typescript to regenerate this file.
+ */
+
+export interface Evaluation {
+  hypercert: {
```

Comment:
Sorry had to leave the discussion early, did we discuss whether we going to continue keeping this pointer?
I don't see the harm, if we can retrieve the metadata of a burned hypercert to get the hypercert region from that?

Reply:
But might be simpler to just have regions and not add complexity

Reply:
yes - agreed!